resources/assets/js/app.js

```diff
@@ -173,7 +173,10 @@ $(document).ready(function () {
                     continue;
                 }
             }
-            data.append(this.elements[i].name, this.elements[i].value)
+            // Add checkboxes only if they are checked
+            if(e.currentTarget.elements[i].type != 'checkbox' || e.currentTarget.elements[i].type == 'checkbox' && e.currentTarget.elements[i].checked) {
```

Comment:
Doesn't it need to be
`if(e.currentTarget.elements[i].type != 'checkbox' || (e.currentTarget.elements[i].type == 'checkbox' && e.currentTarget.elements[i].checked))`
?
Or remove the second `e.currentTarget.elements[i].type == 'checkbox' `
So it gets
`if(e.currentTarget.elements[i].type != 'checkbox' || e.currentTarget.elements[i].checked)`

Reply:
`&&` has higher precedence than `||`, so parentheses aren't actually necessary.  Though @emptynick is right that `type == 'checkbox'` is implied in the right side of the `||`.

Reply:
The "compact" version was applied in my first commit: 74534a3b68a0242c5a535161c29691999d5c6400
The decision is up to you, @fletch3555 !

Reply:
Apologies for the confusion @tschmidiger.  When I suggested `== &&` before, I meant the existing `!=` could be converted, not that you should add what you added.  Looking at it now, I see that would've been incorrect logic, so I'm glad you didn't do that.  Please switch back to the "compact" version.  I'm good with that.

Reply:
Ok, no worries...I just switched back to the first commit.